setup.cfg

```diff
@@ -0,0 +1,51 @@
+[metadata]
+name = audplot
+author = Johannes Wagner, Hagen Wierstorf
+author-email = jwagner@audeering.com, hwierstorf@audeering.com
+url = https://gitlab.audeering.com/tools/audplot/
+project-urls =
+    Documentation = http://tools.pp.audeering.com/audplot/
+description = A Python plotting package
+long-description = file: README.rst, CHANGELOG.rst
+license = MIT License
+license-file = LICENSE
+keywords = example
+platforms= any
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Scientific/Engineering
+
+[options]
+packages = find:
+install_requires =
+    audmetric >=1.0.1, <2.0.0
+    matplotlib
```

Comment:
I think it's sufficient to have `seaborn` as dependency: https://seaborn.pydata.org/installing.html#required-dependencies

Reply:
OK, I removed `matplotlib`, `numpy` and `pandas`